### ISSUE TITLE
Eureka/Raidboss Plugin not doing the Wake Up in Eureka anymore

### ISSUE THREAD

**Issue by ArkfrostLumas** [quisquous/cactbot#295]
I've noticed, that Cactbot isn't doing the Wake Up prompt anymore.
Went into Eureka, tried to write the regex text into the echo chat, didn't work. Waited for the chat message to appear on it's own, didn't work for the prompt. Typed the regex with the numbers from the trigger file before it, worked.

So apparently, SE changed the log value of the wake up prompt, so it isn't 00:0039: anymore.
But I have noticed, it works, completely fine if you simply change

      regex: /00:0039:5 minutes have elapsed since your last activity./,
      regexDe: /00:0039:Seit deiner letzten Aktivität sind 5 Minuten vergangen./,
      regexFr: /00:0039:Votre personnage est inactif depuis 5 minutes/,

to

      regex: /5 minutes have elapsed since your last activity./,
      regexDe: /Seit deiner letzten Aktivität sind 5 Minuten vergangen./,
      regexFr: /Votre personnage est inactif depuis 5 minutes/,

**Comment by quisquous** [comment]
Did this change in the last patch? What does the log message say now?

**Comment by ArkfrostLumas** [comment]
Well, in the chat it still says the same thing as before (because if that had changed I would've changed it in cactbot myself).

However, I don't know, how to find the corresponding number within the logs, because ACT only parses for a minute after my last attack, while the log message pops up after 5 minutes.

**Comment by ArkfrostLumas** [comment]
Ah, I didn't know, that the logs parse that much. Here it is. It's German tho :P 

00|2018-11-30T00:18:58.0000000+01:00|0039||Seit deiner letzten Aktivität sind 5 Minuten vergangen. Wenn du mehr als 10 Minuten inaktiv bist, wirst du automatisch aus dem Spielinhalt entfernt.|c06af83e3857081a01c6ff341a0c39cd

Maybe it is because of the fact that I updated ACT itself (just recently I got my first update program for the software itself in years)?
